src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/email/AppellantForceCaseProgressionToCaseUnderReviewPersonalisationEmailTest.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.appellant.email;

```

Comment:
This file is using CRLF as Line Feed convention. Please change it to LF.

Make sure the same applies for all files involved in this PR.